### PR TITLE
CI build on Windows

### PR DESCRIPTION
**Pull request by tpasternak**
# Checklist

- [ ] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ ] I have received the approval from the maintainers to make this change.
- [ ] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Discussion thread for this change

Issue number: `<please reference the issue number or url here>`

# Description of this change

### CODE REVIEW

File: .bazelci/clion.yml [hunk 1]
```diff
@@ -76,3 +76,12 @@ tasks:
       - //:clwb_tests
     soft_fail:
       - exit_status: 1
+  CLion-OSS-under-dev:
+    name: CLion OSS Under Development
+    platform: ubuntu1804
```
Comment (mai93) [inline .bazelci/clion.yml:81]: should this be `windows`?

File: .bazelci/intellij.yml [hunk 2]
```diff
@@ -76,4 +76,13 @@ tasks:
       - //:ijwb_ce_tests
     soft_fail:
       - exit_status: 1
+  IntelliJ-CE-OSS-under-dev:
+    name: IntelliJ CE OSS Under Development
+    platform: ubuntu1804
```
Comment (mai93) [inline .bazelci/intellij.yml:81]: also this one

**Comment by mai93** [comment]
is there a reason we are only adding builds on windows and not also tests?

**Comment by tpasternak** [comment]
a few tests fail on windows, but having build running there is still better than nothing

**Review by mai93 — APPROVED** [review]

**Pull request merged**